build.sh

```diff
@@ -78,7 +79,7 @@ case $module in
     genOpt CMAKE_BUILD_TYPE=Debug  # build with debugging symbols
     genOpt DELPHES=OFF
     genOpt EVALUATION=OFF
-    genOpt IRT_ROOT_IO=ON
+    genOpt IRT_ROOT_IO=OFF
```

Comment:
Just curious, why do you want this off? We can probably live with it off nowadays though, since `ON` was only needed by Juggler.

Reply:
Sorry! That was actually a mistake while playing with root options, it shouldn't be there.